schema.graphql

```diff
@@ -270,6 +284,27 @@ enum NewsItemsOrder {
   updatedAt_DESC
 }
 
+type OpenWeatherMap {
+  alerts: [String!]
+  createdAt: String
+  current: String
+  daily: [String!]
+  hourly: [String!]
+  id: ID
+  lat: Float
+  lon: Float
+  updatedAt: String
+}
```

Comment:
Gehört es so, dass die types mehrfach definiert sind?

Reply:
das ist eine gute Frage, wir wissen nicht genau warum die identischen Daten immer 2 mal drin sind. bisher hat sich noch keiner getraut, die ein Paar davon zu löschen

Reply:
genau, die schema files liegen einmal in root und einmal in public.
jemand müsste prüfen, was von beiden wovon genutzt wird und ob ein dateipaar dann entfernt werden könnte.
das initiale aufsetzen von graphql serverseitig hatte ein mitarbeiter gemacht, den es nicht mehr gibt.